spyder_unittest/backend/noserunner.py

```diff
@@ -25,6 +25,20 @@ class NoseRunner(RunnerBase):
     module = 'nose'
     name = 'nose'
 
+    def get_versions(self):
+        """Return versions of framework and its plugins."""
+        import nose
+        from pkg_resources import iter_entry_points
```

Comment:
What is this `pkg_resources`? Can we be sure it is installed? I do have it installed but I don't know why.

Reply:
`pkg_resources` comes with `setuptools`, so I guess everyone will have it (see also this [SO question](https://stackoverflow.com/questions/7446187/no-module-named-pkg-resources)).  It is used by `nose` itself too, e.g. [`here`](https://github.com/nose-devs/nose/blob/7c26ad1e6b7d308cafa328ad34736d34028c122a/nose/plugins/manager.py#L378), and we first test whether `nose` is installed before calling `get_versions` in line `201` of the new `spyder_unittest/widgets/unittestgui.py`.